data/mods/MindOverMatter/martial_arts.json

```diff
@@ -0,0 +1,54 @@
+[
+  {
+    "type": "martial_art",
+    "id": "style_gun_kata_precog",
+    "name": { "str": "Mirage Destiny" },
+    "description": "A martial art that requires pre-cognitive abilities.  It allows it's users to strike while on the move and be not where their enemies strike at them.",
```

Comment:
```suggestion
    "description": "A martial art that requires pre-cognitive abilities.  It allows its users to strike while on the move and not be where their enemies strike at them.",
```

Reply:
The "be not" was intentional but I don't feel like altering your correction lol